README.md

```diff
@@ -12,11 +12,11 @@ cd statesman-server/
 
 Change the line on Dockerfile (in the path $CODE/statesman-server/Dockerfile): 
 
-ADD config/docker.yml docker.yml
+ADD config/docker.yml config/docker.yml
 
 To the following for local:
 
-ADD config/local.yml docker.yml
+ADD config/local.yml config/docker.yml
```

Comment:
Actually.. there is a easier way now .. you can volume mound the dir containing the config file and pass the config file path as an env variable called CONFIG_PATH. Check startup.sh file.